clustering/infinispan/extension/src/main/java/org/jboss/as/clustering/infinispan/subsystem/KeyAffinityServiceFactoryBuilder.java

```diff
@@ -54,14 +55,13 @@
  * Returns a trivial implementation if the specified cache is not distributed.
  * @author Paul Ferraro
  */
-public class KeyAffinityServiceFactoryBuilder implements Builder<KeyAffinityServiceFactory>, Service<KeyAffinityServiceFactory>, KeyAffinityServiceFactory {
+public class KeyAffinityServiceFactoryBuilder implements Builder<KeyAffinityServiceFactory> {
 
-    private final String containerName;
-    private volatile int bufferSize = 10;
-    private volatile ExecutorService executor;
+    private final PathAddress address;
+    private volatile int bufferSize = 100;
```

Comment:
Was this change from 10 to 100 deliberate?

I only ask because it came in in a refactoring commit so may have been accidental.